generators/environment/templates/outrigger/outrigger.yml

```diff
@@ -81,8 +108,22 @@ scripts:
     alias: logs
     description: Stream the logs produced by your Docker project containers. To limit output add '--tail=20'.
     run:
+      - if [ ${OSTYPE:0:5} = linux ]; then COMPOSE_FILE=docker-compose.yml; fi
```

Comment:
I forget, what is needed in order to remove the flagging on ${OSTYPE}?

Reply:
We need to finish planning proper & streamlined Linux support: https://docs.google.com/document/d/1Wi9WRgrYD-6Kb8VjH4FjefLqcbU9PLNlwthBZL9yvcs/edit#